elastalert/util.py

```diff
@@ -143,3 +143,31 @@ class EAException(Exception):
 
 def seconds(td):
     return td.seconds + td.days * 24 * 3600
+
+
+def total_seconds(td):
+    # For python 2.6 compatability
+    return (td.microseconds + (td.seconds + td.days * 24 * 3600) * 10 ** 6) / 10 ** 6
```

Comment:
This is taken directly from https://docs.python.org/2/library/datetime.html#datetime.timedelta.total_seconds